muesli/web/templates/Fragments/main.pt

```diff
@@ -10,6 +10,7 @@
   <title>MÜSLI</title>
   <link rel="stylesheet" type="text/css" tal:attributes="href request.static_url('muesli.web:static/css/muesli.css')"></link>
   <link rel="stylesheet" type="text/css" tal:attributes="href request.static_url('muesli.web:static/css/fancybox/jquery.fancybox.css')"></link>
+  <link rel="stylesheet" type="text/css" tal:attributes="href request.static_url('muesli.web:static/css/select2.min.css')"></link>
```

Comment:
There is no method to load only specific css-files. THere is one for js-files wich could easily be adapted. This was not done for now, because this should also be used for jquery.fancybox.css in the old code.
If you want I will add this functionality.

Reply:
No, that's alright. I just wanted to ask, if there is a reason for this. Since it reduces complexity in some way, that's enough for me :-)